xbmc/addons/AddonManager.h

```diff
@@ -117,6 +117,8 @@ namespace ADDON
 
     bool GetInstallableAddons(VECADDONS& addons, const TYPE &type);
 
+    bool GetInstalledBinaryAddonPaths(std::vector<std::pair<std::string, bool>>& binaryAddonList);
```

Comment:
Could you please add documentation for the new member function? Especially the content of the vector is not self explaining.